Cluster quick install (#1009)
temporalio/documentation: docs/devtools/web-ui.md

## Using Temporal Web for development

> Once you have the Temporal Server running locally (use the [quick install guide](/docs/clusters/quick-install)), you can view the Temporal Web UI at [http://localhost:8088](http://localhost:8088) (this is configurable with the `TEMPORAL_WEB_PORT` environment variable).

Reviewer: ```suggestion
When you have the Temporal Server running locally (use the [quick install guide](/docs/clusters/quick-install)), you can view the Temporal Web UI at [http://localhost:8088](http://localhost:8088).
(You can configure the port number by setting the `TEMPORAL_WEB_PORT` environment variable.)
```
